### ISSUE TITLE
What is the ideal loss value?

### ISSUE THREAD

**Issue by offchan42** [thtrieu/darktf#758]
What is the ideal loss value for `darkflow` during training?
Because there is no option to evaluate loss on a validation set, knowing this value should be useful.

Is it similar to this repository?
https://github.com/AlexeyAB/darknet
It stated that we should stop training when the average loss is around `0.060370`.
But I'm skeptical because I suspect that the loss calculation mechanism might be different.

**Comment by off99555** [comment]
I got 0.3 loss using `darknet` and the mAP was about 50% to 60%. So it was quite good for me.
Maybe the loss is dataset-specific. I don't know.

**Comment by Malouke** [comment]
hi off99555,
you can give us more details pls .
how many images learning rate epochs batch subdivision....
thx

**Comment by off99555** [comment]
I got 4,000 images, everything else is set to default.
I had done the training job a long time ago.

**Comment by TranDinhKhang** [comment]
> I trained yolov2.cfg on my own dataset for object detection and I got some OK predictions(about 50% accuracy). The avg loss stopped decreasing at around 2.
> I trained yolov2-voc.cfg on the Pascal VOC dataset with the initialization of pre-trained weights(yolov2-voc.weights) and the avg loss stopped decreasing around 3.
> I didn't look at the code but I think you are right that loss calculation of darkflow is different from darknet.

yes, me too. my loss calculated around 1.6 to 2.5
